app/renderer/actions/Session.js

```diff
@@ -222,7 +222,7 @@ export function newSession (caps, attachSessId = null) {
       }
       case ServerTypes.perfecto:
         host = session.server.perfecto.hostname;
-        port = session.server.perfecto.port;
+        port = session.server.perfecto.port || 80;
```

Comment:
should it be `?? 80` or zero value is also invalid?

Reply:
0 should also be invalid in this case.